Bug: capital_pct strategy runs with /bin/zsh capital when balance fetch fails

## Description

In `scheduler/balance.go` around line 83, when `resolveCapitalPct` fails to fetch a platform balance and no fallback `capital` is set, the strategy proceeds with `Capital <= 0`. Config validation (`config.go` line 346) allows `capital=0, capital_pct>0`, so this is a supported config that will silently run with zero capital.

## Fix

Add a guard in the per-strategy loop after `resolveCapitalPct`:
```go
if sc.CapitalPct > 0 && sc.Capital <= 0 {
    fmt.Printf("[ERROR] %s: capital_pct set but capital resolved to $0 — skipping\n", sc.ID)
    continue
}
```

## Origin

Found during code review of PR #89 (capital_pct). Confidence: 85.

--- a/scheduler/main.go
+++ b/scheduler/main.go
@@ -253,6 +253,12 @@ func main() {
 		// Determine which strategies are due this tick
 		dueStrategies := make([]StrategyConfig, 0)
 		for _, sc := range cfg.Strategies {
+			// #100: Skip strategies where capital_pct is set but capital resolved to $0
+			// (balance fetch failed and no fallback capital configured).
+			if sc.CapitalPct > 0 && sc.Capital <= 0 {
+				fmt.Printf("[ERROR] %s: capital_pct set but capital resolved to $0 — skipping\n", sc.ID)
+				continue
+			}
 			interval := sc.IntervalSeconds
 			if interval <= 0 {
 				interval = cfg.IntervalSeconds

